src/main.js

```diff
@@ -153,11 +154,31 @@ app.on('ready', () => {
       })
 
       chain.on("stderr", (data) => {
-        mainWindow.webContents.send(ADD_LOG_LINES, data.split(/\n/g))
+        const lines = data.split(/\n/g)
+        mainWindow.webContents.send(ADD_LOG_LINES, lines)
+
+        const errorPattern = /Error: /g
+        const stackPattern = /^    at /g
+        for (let i = 0; i < lines.length; i++) {
+          if (errorPattern.exec(lines[i])) {
+            // check if the next line has a stack
+            if (i === lines.length - 1 || stackPattern.exec(lines[i+1])) {
+              // either it's the last line that has an error or there is an error
+              //   with a call stack trace, lets keep track of this in case we crash
+              logError = lines[i]
+              if (i < lines.length - 1) {
+                logError += lines[i+1]
+              }
+            }
+          }
+        }
```

Comment:
Readability: (L157-L174) likely belongs in its own helper function.

Also, what's it for? If we want to get stack traces for error messages they should be included in the error object sent from the chain process via IPC, not parsed out of the logs.